pkg/util/nvidia/paths.go

```diff
@@ -60,11 +60,10 @@ func nvidiaContainerCli(args ...string) ([]string, error) {
 }
 
 // nvliblist returns libraries listed in nvliblist.conf which is located in nvidiaDir.
```

Comment:
Should you update the comment since now file path is accepted, not directory?

Reply:
ah, thanks